tools/core_packages.txt

```diff
@@ -1,74 +1,104 @@
-acl
-attr
-binutils
 brotli
 bz2
-c_ares
 ca_certificates
+c_ares
 crew_profile_base
-curl
+docbook_xml
+docbook_xml412
+docbook_xml42
+docbook_xml43
+docbook_xml44
+docbook_xml45
+docbook_xml50
+docbook_xml51
+docbook_xsl
+docbook_xsl_nons
 e2fsprogs
 elfutils
 expat
 filecmd
 flex
-gdbm
+gcc
 gcc11
+gdbm
 gettext
 git
 glibc
+gmp
 gnutls
 groff
 icu4c
+isl
+jansson
+jemalloc
 krb5
 less
 libarchive
+libcurl
 libcyrussasl
 libdb
 libedit
 libffi
+libgcrypt
+libgpgerror
 libidn2
 libmetalink
+libmicrohttpd
 libnghttp2
 libpipeline
 libpsl
 libseccomp
 libsigsegv
+libssh
 libssp
 libtasn1
 libtirpc
+libunbound
 libunistring
 libxml2
+libxslt
 libyaml
+llvm
 lz4
 lzip
 m4
 mandb
 manpages
 mawk
 most
+mpc
+mpdecimal
+mpfr
+musl_zstd
 ncurses
 nettle
 openldap
+openmp
 openssl
 p11kit
 pcre
 pcre2
 perl
 pixz
+popt
+py3_libxml2
+py3_packaging
 py3_pip
 py3_setuptools
 py3_wheel
 python2
 python3
 readline
+rsync
 rtmpdump
 ruby
-sed
```

Comment:
This should be core.  The out-of-the-box sed is worthless.

Reply:
AFAIK `sed` was moved away to `buildessential` times ago (only `patch` and `build` section needs it)

Reply:
And how many packages use it?  About half?  I think we should add it back.

Reply:
After further thought, it's fine to leave this in buildessential.  I think we should move llvm to buildessential, if possible.  That should NOT be in core.

Reply:
> After further thought, it's fine to leave this in buildessential. I think we should move llvm to buildessential, if possible. That should NOT be in core.

As soon as we can break out the llvm libraries into another package!

Reply:
Requiring llvm in core is cringeworthy.